R/mod_quality_overview.R

```diff
@@ -270,4 +286,32 @@ mod_quality_overview_server <- function(input,
         unique()
       })
    })
+  #filtering by achievement levels
+  shiny::observe({
+    clicked_level <- list(F, F, F)
+    names(clicked_level) <- c("legend_high", "legend_mod", "legend_low")
+    filter_indicator$level <- list("H", "M", "L", "undefined")
+
+    level_buttons <- lapply(
+      names(clicked_level),
+      function(button) {
+        shiny::observeEvent(
+          shiny::req(input[[button]]), {
+            if (clicked_level[[button]]) {
+              clicked_level <<- list(F, F, F)
+              names(clicked_level) <<- c("legend_high", "legend_mod", "legend_low")
+              filter_indicator$level <- list("H", "M", "L", "undefined")
```

Comment:
These lines are not tested. Is it possible to add test?

Reply:
I dont know how to do it. That part is activated when the buttons are clicked for the second time.

Reply:
It seems to work now with f869f5b. With this:
```R
session$setInputs(legend_high = 1)
session$setInputs(legend_high = 1) # twice
session$setInputs(legend_mod = 0)
session$setInputs(legend_low = 0)
expect_equal_to_reference(output$qi_table, "data/output_qi_table_filter_level_high_2017.rds")
```